### DISCUSSION TITLE
Notifier: How to wait for noice before propagating, similar to lazy_notify in lazyvim

### DISCUSSION THREAD

**Discussion by GNRSN** [folke/snacks.nvim#225] [Q&A] [answered]
When using Noice and nvim-notify I've had my own version of [lazy_notify from lazyvim ](https://github.com/LazyVim/LazyVim/blob/71ea193cf6ee9e8021be2e8c7f277f757779c1ac/lua/lazyvim/util/init.lua#L117). Since I dont want to wait with loading snacks until after Noice, i'm not getting this to work so I'm seeing a deprecation warning on startup that I'm otherwise blocking through Noice. 

Any suggestions for how to achieve similar behavior?

**Comment by folke** [comment]
You can check how it's done in lazyvim. lazy_notify and also check `lua/lazyvim/plugins/init.lua` at the bottom

**Comment by GNRSN** [comment, marked as answer]
Thank you for the pointer, I had not seen the snacks setup call in LazyVim. 

If anybody else is interested, [here is the resulting diff](https://github.com/GNRSN/dotfiles/commit/1bc66c78c14ff89a70570585558db04fc6b30bce#diff-219f2e0a744e665d0a2fd5002096e231bc4c5f06d5f2d56903f74ef0671222b1R73)
